app/views/catalog/_home_text.html.erb

```diff
@@ -40,4 +40,7 @@
       <%= content_tag :div, '', id: 'map', aria: { label: t('geoblacklight.map.label') }, data: { map: 'home', 'catalog-path'=> search_catalog_path , 'map-bbox' => params[:bbox], basemap: geoblacklight_basemap, leaflet_options: leaflet_options } %>
     </div>
   </div>
+  <div align='right'>
+  <%= content_tag :h5, t('geoblacklight.home.version') %> <%= Geoblacklight::VERSION %></p>
```

Comment:
Using an `h5` here gives us a heading structure problem. 

![Screen Shot 2020-08-04 at 9 45 06 AM](https://user-images.githubusercontent.com/1656824/89315557-3988ca00-d638-11ea-872f-30e330c96bf3.png)


Also lets add the version as a variable in the i18n key so that adopters could easily remove this.

```suggestion
    <%= t('geoblacklight.home.version', version: Geoblacklight::VERSION ) %>
```